__tests__/integration/verifyPreviewRDF.test.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import pupExpect from 'expect-puppeteer'
+import { testUserLogin } from './loginHelper'
+import { fillInRequredFieldsForBibframeInstance } from './previewRDFHelper'
+
+describe('Previewing the RDF', () => {
+  beforeAll(async () => {
+    return await testUserLogin()
+  })
+
+  beforeEach(async () => {
+    return await fillInRequredFieldsForBibframeInstance()
+  })
+
+  it('builds the rdf and verifies the expected content', async () => {
+    expect.assertions(6) // An additional assertion is done in previewRDFSetup
```

Comment:
Change comment to read `fillInRequredFieldsForBibframeInstance`

Reply:
Done.